src/tests/system/tests/test_kcm.py

```diff
@@ -413,3 +412,4 @@ def test_kcm__simple_kinit(client: Client, kdc: KDC):
             with pytest.raises(SSHProcessError):
                 krb.kinit(username, password="wrong")
             assert krb.klist().rc == 0, "Klist failed"
+
```

Comment:
I'm not sure why but flake8 and pycodestyle are complaining about a blank line at the end of the file.  I assume this was added to avoid the "no newline at the end of file" type errors?

Reply:
They were two lines.